src/typography/Text/Text.stories.tsx

```diff
@@ -24,6 +24,10 @@ export default {
             options: ['default', 'weak', 'x-weak', 'disabled', 'negative', 'positive', 'warning', 'interactive'],
             control: { type: 'select' },
         },
+        boxColor: {
+            options: [undefined, 'neutral', 'selected', 'disabled', 'positive', 'negative', 'warning'],
```

Comment:
When you put undefined as an option, it shows as "Undefined" on the dropdown. I think it will look better to have an empty string. It will interpret it as undefined.